modules/access_outages/js/affinity_group_outages.js

```diff
@@ -125,7 +96,7 @@ const showCurrentOutages = async function showCurrentOutages(ciderIds, bDebugWit
  */
 function getOutageHtml(outage) {
   return `
-    <a href="https://support.access-ci.org/outages?outageID=` + outage['ID'] + `">
+    <a style="text-decoration: none" href="/outages?outageID=` + outage['URN'] + `">
```

Comment:
i was seeing underlines for the link -- but should this get moved to a style sheet?

Reply:
Good enough.